### DISCUSSION TITLE
The Cookie value is showing incorrect category for some of the cookies

### DISCUSSION THREAD

**Discussion by prakashkatariyatatvic** [GoogleChromeLabs/ps-analysis-tool#641] [Support Forum] [answered]
While performing audit via privacy sandbox extension on google chrome, we have observed that for some of the cookies the cookie category is not showing correct value.  I have attached the screenshot for your reference below:

**Screenshot Link:** http://prntscr.com/OhdySbNjTqpf
![image](https://github.com/GoogleChromeLabs/ps-analysis-tool/assets/157360089/f63fd712-8730-4fc0-bda3-10d10527d578)

Google Chrome Version: v120

Please let us know if we have any solution to get correct value every time or anyone found any alternatives to avoid this !!

Thanks in advance!!
Prakash

**Comment by milindmore22** [comment]
Hi @prakashkatariyatatvic,

Thanks for catching this! You're right, the `AnalyticsSyncHistory` cookie should be categorized under Analytics rather than Functional as listed on the LinkedIn [cookie page](https://www.linkedin.com/legal/l/cookie-table#:~:text=Reddit-,Analytics%20and%20Research,-Purpose%3A%20Analytics%20and).

I've opened a PR (https://github.com/rtCamp/Open-Cookie-Database/pull/8) to correct the category, and the fix will be available in the upcoming release of PSAT extension.

**Comment by prakashkatariyatatvic** [comment]
Hi @milindmore22 

Thanks for quick response and support

Can you please help me understand by when can we get stable version of it ? and by when can we completely rely on this to check and compare all cookies ?

apart from this, just wanted to highlight that we are not able to copy all the details mentioned in extension, can you please allow user to copy all the details of cookie ? it would be great if you allow this option so that we can copy all the details and share it with others!!

Thanks in advance

**Comment by milindmore22** [comment, marked as answer]
@prakashkatariyatatvic thank you for your patience 

We are actively working towards the stable version release, while the exact date is still being finalized, your input is crucial to us. Please don't hesitate to report any issues or queries you encounter, as this will help us keep evolving towards the stable release. We'll provide updates as soon as we have a more precise timeline.

Additionally, we would like to mention that we leverage the [Open Cookie Database](https://github.com/jkwakman/Open-Cookie-Database), which continuously evolving thanks to community contributions.

Furthermore, we are actively developing a feature ([#228](https://github.com/GoogleChromeLabs/ps-analysis-tool/issues/228)) that facilitates the export of all cookies. Meanwhile, you can make use of the PSAT CLI, which conveniently exports cookie details in both JSON and CSV formats, ensuring well-organized and clean data. For detailed instructions, please refer to our [GitHub wiki](https://github.com/GoogleChromeLabs/ps-analysis-tool/wiki/PSAT-Command-Line-Interface).

**Comment by milindmore22** [comment]
@prakashkatariyatatvic I will be converting this issue to support forum discussion, For future reference, please [create support discussion](https://github.com/GoogleChromeLabs/ps-analysis-tool/discussions/new?category=support-forum) if you have any question about the PSAT.
